developer/src/kmc/src/commands/build.ts

```diff
@@ -27,6 +27,12 @@ export function declareBuild(program: Command) {
 async function build(infile: string, options: BuildActivityOptions): Promise<boolean> {
   console.log(`Building ${infile}`);
 
+  if(!fs.existsSync(infile)) {
+    // TODO: consolidate errors
+    console.error(`File ${infile} does not exist`);
+    process.exit(2);
```

Comment:
Is there documentation on why the exit code of 2 was selected?

Reply:
No, but will be considered with error message cleanup.